src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
+  let newState = {};
```

Comment:
The declaration of `newState` here is redundant since you're assigning it a new value in each case of the switch statement. Consider removing this line.